### ISSUE TITLE
Site replaced

### ISSUE THREAD

**Issue by Floppy** [Data-Liberation-Front/csvlint.io#773]
@davetaz I see that https://csvlint.io has completely changed to https://github.com/theodi/csvlint.io, so I assume this repo should be archived?

**Comment by davetaz** [comment]
@Floppy Yes. I built a new version. It still uses the gem which I've also made some changes too and sent you a PR. 

https://github.com/Data-Liberation-Front/csvlint.rb/pull/299

**Comment by Floppy** [comment]
Ah, it still uses the gem behind the scenes? OK, good to know. I'll archive this frontend then, and get those changes merged in.
